### PR TITLE
Nov 2016 monthly update

### PR DESCRIPTION
**Pull request by indirect**

### CODE REVIEW

File: app/views/news/2016-11-30-november-2016-monthly-update.html.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+summary: "This month 16 new members joined, André spoke at RubyConf about contributing to open source, and we paid for 129.5 hours of work on Bundler, RubyGems, and RubyGems.org."
+---
+
+<% title "November 2016 Monthly Update" %>
+
+Welcome to the November 2016 Ruby Together monthly update! This month included maintenance work, security patches, helping users with problems, and [@segiddins](https://github.com/segiddins)'s heroic (but ultimately doomed) efforts to make Bundler respect `required_ruby_verison`.
```
Comment (segiddins) [inline app/views/news/2016-11-30-november-2016-monthly-update.html.md]: wait, what?

File: app/views/news/2016-11-30-november-2016-monthly-update.html.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+
+## bundler news
+
+This month bundler and [bundler.io](bundler.io) got an array of documentation fixes / updates, due in large part to our new bundler contributor [@colby-swandale](https://github.com/colby-swandale).
+
+Bundler got a [Request For Comments](https://github.com/bundler/rfcs) repo, inspired by the one utilized by [Rust](https://www.rust-lang.org/en-US/). [Fixed](https://github.com/bundler/bundler/pull/5176) outdated not listing all outdated gems. And [fixed](https://github.com/bundler/bundler/pull/5213) an `ObjectBoundsExceededError` with [Rubinius](https://rubinius.com/).
+
+As mentioned last month, Bundler 1.13 shipped with [`required_ruby_version` support](http://bundler.io/blog/2016/09/08/bundler-1-13.html) for Gemfiles containing a `ruby` declaration. Building on that, we hoped to release 1.14 with automatic support for the currently running Ruby version.
```
Comment (segiddins) [inline app/views/news/2016-11-30-november-2016-monthly-update.html.md]: we dropped this from 1.14's roadmap

File: app/views/news/2016-11-30-november-2016-monthly-update.html.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+While @segiddins [nobly attempted](https://github.com/bundler/bundler/pull/5013) to implement it, we ran into a plethora of edge cases and problems with backwards compatibility. We'll aim to post a longer writeup of the situation on [the Bundler blog](https://bundler.io/blog) soon.
+
+In the meantime, the workaround for now is to put `ruby RUBY_VERSION` in your `Gemfile`. Now that we've concluded that automatic Ruby version management needs to wait for Bundler 2.0, we'll be able to release 1.14 very soon.
```
Comment (segiddins) [inline app/views/news/2016-11-30-november-2016-monthly-update.html.md]: I don't think this is good advice, I'd prefer people put an explicit version in their Gemfiles

Comment (parndt) [inline app/views/news/2016-11-30-november-2016-monthly-update.html.md]: I don't understand how it makes sense having the line if it's just going to equal whatever version is running?

**Pull request merged**